pkg/config/kind.go

```diff
@@ -85,14 +85,17 @@ func (ks KindSet) Set(k Kind) KindSet {
 	return ks | (1 << k)
 }
 
+// make gas happy
+func ignoreErrors(args ...interface{}) {}
+
 func (ks KindSet) String() string {
 	buf := pool.GetBuffer()
 	defer pool.PutBuffer(buf) // fine to pay the defer overhead; this is out of request path
 
-	fmt.Fprint(buf, "[")
+	ignoreErrors(fmt.Fprint(buf, "["))
```

Comment:
```
pkg/aspect/metricsManager.go:160:22:warning: unnecessary conversion (unconvert)
pkg/aspect/quotasManager.go:167::error: arg name for printf verb %s of wrong type: int (vet)
pkg/config/kind.go:92::warning: Errors unhandled.,LOW,HIGH (gas)
pkg/config/kind.go:95::warning: Errors unhandled.,LOW,HIGH (gas)
pkg/config/kind.go:105::warning: Errors unhandled.,LOW,HIGH (gas)
```
This is after ensuring `prep_linters` does all the `go gets`.
We need to pin linters to a version.